io.github.realkleiner.yafc-ce.yml

```
@@ -0,0 +1,51 @@
+app-id: io.github.realkleiner.yafc-ce
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  prepend-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+  prepend-pkg-config-path: "/usr/lib/sdk/dotnet8/lib/pkgconfig"
+command: Yafc
+finish-args:
+  - "--device=dri"
+  - "--socket=x11"
+  #- "--share=ipc"
```

```suggestion
  - "--share=ipc"
```

it's in the doc. It required for x11.